docs/ops/detection/RegionYolo_1.md

```diff
@@ -6,7 +6,7 @@
 
 **Short description**: *RegionYolo* computes the coordinates of regions with probability for each class.
 
-**Detailed description**: This operation is directly mapped to the original YOLO layer. [Reference](https://arxiv.org/pdf/1612.08242.pdf)
+**Detailed description**: This operation is directly mapped to the [original YOLO layer](https://arxiv.org/pdf/1612.08242.pdf). 
```

Comment:
I think that we have rule of thumb that the text in the square brackets should be equal to the title of the link. So in our case it should look like:

> ... directly mapped to the [YOLO9000: Better, Faster, Stronger](https://arxiv.org/pdf/1612.08242.pdf) paper.

Reply:
Changed in this [commit](https://github.com/openvinotoolkit/openvino/pull/5926/commits/c0fd5d8d5baf158ea39d4f88df108dd61c530496).